dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/AuditData.java

```diff
@@ -34,11 +34,16 @@
                                             LogFieldConstants.THREADID,
                                             LogFieldConstants.HOSTNAME,
                                             LogFieldConstants.WLPUSERDIR,
-                                            LogFieldConstants.SERVERNAME
+                                            LogFieldConstants.SERVERNAME,
+                                            LogFieldConstants.TYPE
     };
```

Comment:
Originally NAMES1_1 referred to the set of names used for JSON and logstashCollector-1.1 (which we got rid of), and NAMES referred to the set of names used for logstashCollector-1.0.  Can you please rename NAMES1_1 / NAMES to NAMES_JSON / NAMES_LC (across all of these XXXData.java files)?